src/iqm/qiskit_iqm/iqm_transpilation.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2023 Qiskit on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Transpilation tool to optimize virtual Z rotations tailored to IQM hardware."""
```

Comment:
I think we should not use the term "virtual" anywhere in this optimizer. These are real Z rotations in the circuit, we just optimize them away. The word "virtual" refers to some  implementation of Rz gate on control instrument level.

Reply:
True, removed